sklearn/naive_bayes.py

```diff
@@ -460,6 +463,16 @@ def _update_class_log_prior(self, class_prior=None):
         else:
             self.class_log_prior_ = np.zeros(n_classes) - np.log(n_classes)
 
+    def _check_alpha(self):
+        if self.alpha < 0:
+            raise ValueError('Smoothing parameter alpha = %.1e. '
+                             'alpha must be >= 0!' % self.alpha)
```

Comment:
Just noticed this says >= which is a little awkward we warn when it's 0 and change the value